playkit/src/main/java/com/kaltura/playkit/plugins/KalturaStatsPlugin.java

```diff
@@ -179,6 +183,12 @@ protected void onUpdateMedia(PKMediaConfig mediaConfig) {
         } else {
             baseUrl = DEFAULT_BASE_URL;
         }
+        if (Utils.isJsonObjectValueValid(pluginConfig, "timerInterval")) {
```

Comment:
Maybe it's a good time to create a Config class for this plugin. Then the app knows what to set.